man/systemd-vmspawn.xml

```diff
@@ -34,6 +34,8 @@
     <para><command>systemd-vmspawn</command> may be used to start a virtual machine from an OS image. In many ways it is similar to <citerefentry
     project='man-pages'><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>, but it
     launches a full virtual machine instead of using namespaces.</para>
+
+    <para>Note: systemd-vmspawn requires the user to be in the <literal>kvm</literal> group to use the VSock options.</para>
```

Comment:
This is only required on Ubuntu/Debian and derivatives. On Fedora it will work fine without being in the kvm group.

Reply:
Fixed in ef3bb39e80.